Chatty Cathy/Chatty Cathy/TextView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  TextView.swift
+//  Chatty Cathy
+//
+//  Created by Artesia Ko on 2/18/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class TextView: UIView {
+    // MARK: View vars
+    var textBox: UITextView = {
+       let textView = UITextView()
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.isEditable = false
+        textView.showsVerticalScrollIndicator = false
+        textView.font = UIFont.systemFont(ofSize: 12, weight: .regular)
+        textView.textColor = .black
+        return textView
+    }()
+
+    // MARK: View initialization
+    override init(frame: CGRect) {
+        super.init(frame: frame)
```

Comment:
Make sure to add these subviews to this view so they would show up when rendered.